api/src/pcapi/sandboxes/scripts/creators/industrial/create_industrial_bookings_for_statistics.py

```diff
@@ -0,0 +1,290 @@
+from datetime import datetime
+from datetime import timedelta
+import logging
+
+from pcapi.core.bookings.factories import BookingFactory
+from pcapi.core.bookings.models import BookingStatus
+import pcapi.core.educational.factories as educational_factories
+from pcapi.core.educational.models import CollectiveBookingStatus
+import pcapi.core.finance.factories as finance_factories
+import pcapi.core.offerers.factories as offerers_factories
+import pcapi.core.offerers.models as offerers_models
+import pcapi.core.offers.factories as offers_factories
+from pcapi.core.offers.models import Offer
+from pcapi.repository import repository
+
+
+logger = logging.getLogger(__name__)
+
+
+def create_industrial_bookings_for_statistics() -> None:
+    logger.info("create_industrial_bookings_for_statistics")
+    _create_booking_for_statistics_offerer_multiple_venues()
+    _create_booking_for_statistics_offerer_one_venues()
+    _create_booking_for_statistics_individual_only()
+    _create_booking_for_statistics_collective_only()
+    _create_booking_for_statistics_no_offer_no_revenue()
+    _create_booking_for_statistics_with_offer_no_revenue()
+    logger.info("created bookings for statistics")
+
+
+def _create_booking_for_statistics_offerer_multiple_venues() -> None:
+    offerer_multi_venues = offerers_factories.OffererFactory(name="Stats - plusieurs lieux - CA et CA prévisionnel")
+    user_offerer = offerers_factories.UserOffererFactory(offerer=offerer_multi_venues, user__email="api@example.com")
```

Comment:
```suggestion
    user_offerer = offerers_factories.UserOffererFactory(offerer=offerer_multi_venues, user__email="retention_structures@example.com")
```

me semble plus pertinent